### PR TITLE
Create code of conduct

### PR DESCRIPTION
**Pull request by MaxThevenet**
Contributor Covenant Code of Conduct

### CODE REVIEW

**Comment by MaxThevenet** [comment]
The push tests fail (dunno why), but the PR build passes, so I think this is ready for merging.

**Comment by ax3l** [comment]
We are investigating if we can take a more institutional e-mail address as contact.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at warpx-coc@lbl.gov. All
```
Comment (ax3l) [inline CODE_OF_CONDUCT.md]: Note: In it's initial setup, @MaxThevenet, @RemiLehe, @LDAmorim, @ax3l, and @jlvay will receive e-mails send to this address. (Please accept the invite you got.)

**Comment by ax3l** [comment]
Thanks for participating, Diana! CoC text credits go to https://www.contributor-covenant.org :)

**Comment by MaxThevenet** [comment]
Thanks @ax3l !

**Pull request merged**